code/modules/research/techweb/all_nodes.dm

```diff
@@ -14,6 +14,7 @@
 		"basic_matter_bin",
 		"basic_micro_laser",
 		"basic_scanning",
+		"basic_manipulator",
```

Comment:
ABC order

Reply:
Oh yeah I was gonna correct that but then I forgot

Reply:
It's mostly my fault for not setting up linters on this yet 😅